### ISSUE TITLE
Logos 2.0

### ISSUE THREAD

**Issue by jdorfman** [opensourcedesign/logos#5]
There is a great Open Source project called [instant-logos](https://github.com/kogg/instant-logos) that allows you to search (e.g. [Linux](http://instantlogosearch.com/?q=linux)) for basically any logo and you then download a png or svg. I suggest we fork it for the following reasons:
- **Only** allow Open Source logos
- License for the logo (e.g. `CC BY 3.0`)
- Copyright Holder (e.g. Acme Software Foundation)
- Designer (e.g. [Jason Long](https://twitter.com/jasonlong))
- Make it work on GitHub Pages (less maintenance)
- Replace Ads with links to donate to (_fill in the blank_) Foundation

We can also just partner with Instant Logos.

Thoughts?

cc: @auremoser

**Comment by simonv3** [comment]
Hmm, I was going to suggest joining forces but it looks like there's no way to easily contribute to the actual website, which is how I would suggest adding those extra details.

**Comment by jdorfman** [comment]
@simonv3 am I being too ambitious?

**Comment by simonv3** [comment]
@jdorfman I don't think so! I think this could also very easily be powered with just a jekyll site running on GitHub where the repository with the images is the canonical truth and there's a pretty front-end. It's just a shame that we can't make those tweaks to the original instant-logos site. Though maybe the author is down for talking? I don't know them, so I'm hesitant to @ mention them in this thread.

**Comment by jdorfman** [comment]
@simonv3 I will reach out

**Comment by elioqoshi** [comment]
That's a great idea! Count me in!
Also, let's make sure the logos are added in SVG format, properly minified.

**Comment by jdorfman** [comment]
Still waiting to hear back

**Comment by auremoser** [comment]
Thanks @jdorfman sorry for the delay. Not sure what you've all decided but I can bring someone from Fairpixels (see #4) into the thread if you'd like, they spoke on a Mozilla Community Call. 

I'm fine with helping to combine with Instant though, looks rad. But if you want to go the gh-pages route we could probably do something simple and static with cards...like [Airport Codes](http://airportcod.es/#) or [FlatUI colors](https://flatuicolors.com/) where we just tile the logos and make them available for download. Maybe we do a 'best of' site with the tope 20, every month or so, pulling from the most recent additions to all these sources.

**Comment by jdorfman** [comment]
@auremoser I really dig Airport codes, especially since the search engine uses all metadata.

Any others you recommend?

**Comment by jdorfman** [comment]
>  Not sure what you've all decided but I can bring someone from Fairpixels (see #4) into the thread if you'd like, they spoke on a Mozilla Community Call.

More the merrier. =)

**Comment by saiichihashimoto** [comment]
Hey yall! @cameronrohani, @marcomarandiz and I made Instant Logo Search. We made it and, after awhile, moved on to other things. But we still have much love for it and would LOVE to see people contribute to it. Sorry for the delay.

[instant-logos](https://github.com/kogg/instant-logos) is a repository for the logos. [InstantLogoSearch](https://github.com/kogg/InstantLogoSearch) is the actual website. The latter uses the former and a lot of the implementation isn't best (or if having separate repos was a good idea at all), but we made it as quickly as we could. There's a huge laundry list of things we'd love to do (use webpack, host straight from github pages, TONS more logos, etc). It's currently a mess, but it shouldn't be difficult to clean up.

We thought about adding licensing, copyright, designs, etc. and we decided _not_ to do that. Websites that serve a similar use case are usually **super cluttered**. We don't have a million ads or a bunch of the stuff they do, but we took the opposite extreme of having the meanest cleanest instant logo machine. So we stripped EVERYTHING that wasn't that. Even now, if we were to redo it, we may have erred on not doing the bucket functionality and maybe adding it when it seemed necessary. We'd like to keep it that simple.

We want it to stay clean, simple, and very single purposed.
We also want the listing to be as expansive as possible.
We also see your use case as valuable and it would be awesome to support it, somehow.

Something that does those things AND encourages collaboration is what I'd say we aim for. Maybe have [instant-logos](https://github.com/kogg/instant-logos) and then add links on [InstantLogoSearch](https://github.com/kogg/InstantLogoSearch) to the github pages? Something.

I'll let @cameronrohani & @marcomarndiz chime in here. Seeing InstantLogoSearch live on would be awesome.

**Comment by marcomarandiz** [comment]
I'm with @saiichihashimoto on this one. Simple is key - it's what made us so widely successful initially. 
I'd like to hear how this open source version of Instant Logo Search logos and metadata might be different from ClearBit. It would give more context for this continued conversation.

**Comment by jdorfman** [comment]
@saiichihashimoto @marcomarandiz I (now) agree with you regarding keeping it simple. Plus Wikipedia already does an amazing job of keeping that information, which would just be overlap:

![image](https://cloud.githubusercontent.com/assets/398230/19489301/88729b5e-951f-11e6-97e0-6520502c87dd.png)

_via:_ https://commons.wikimedia.org/wiki/File:Gnu-30-banner-without-background.svg

Regarding [ClearBit](http://blog.clearbit.com/logo/), I totally forgot about that service. Maybe we are trying to reinvent the wheel or going down the [xkcd/927](http://xkcd.com/927/) road:

![image](https://cloud.githubusercontent.com/assets/398230/19489433/199381d4-9520-11e6-99ed-a4eeaa68b020.png)

Need to rethink this one.

**Comment by saiichihashimoto** [comment]
xkcd, making our problems relatable. :-)

I just want to reiterate: It would be amazing to have your logos and our logos not just overlap but be the same. Having your work just show up on instantlogosearch and vice versa would be amazing. If there's work needed in our repos to get that going, let's discuss it.

**Comment by jdorfman** [comment]
@saiichihashimoto I think we would just need two things for now
- a way of adding logos (the [readme](https://github.com/kogg/instant-logos/blob/develop/README.md) doesn't have any directions)
- a reserved search-term/uri `osd` or `open-source` to show all Open Source related logos

cc: @marcomarandiz

**Comment by jdorfman** [comment]
@saiichihashimoto what do you think? ⬆️

**Comment by bnvk** [comment]
Hey all. Great thread + idea. Thanks @jdorfman for starting on this- it's 
precisely one of the useful tools / resources I was envisioning when I started 
this repo :smile:

Big hat tip to @saiichihashimoto @@cameronrohani @marcomarandiz :wink:

Would you see this existing at http://opensourcedesign.net/logos or perhaps
/resources/logos or something similar?

**Comment by jdorfman** [comment]
@bnvk to avoid overlap & maintenance I think partnering with [ILS](http://instantlogosearch.com) instead of maintaining our own would be ideal. With that said improvements are needed to ILS. Right now it isn't clear how logos can be added and [one method is broken](https://github.com/kogg/instant-logos/issues/4357).

And their communication is slow... ;)

**Comment by bnvk** [comment]
> @bnvk to avoid overlap & maintenance I think partnering with ILS instead of 
> maintaining our own would be ideal. 

I get that line of thinking and it's totally valid :smile: I also see benefit 
to the approach of merging applicable things, such as...

> And their communication is slow... ;)

In this case, ILS would benefit from belonging to a larger community with
more contributors to lend a hand, etc...

Anywho, up for the existing maintainers and you to figure out :grin:

**Comment by saiichihashimoto** [comment]
https://github.com/kogg/instant-logos/issues/4359

**Comment by jdorfman** [comment]
Could @bnvk @saiichihashimoto and whoever do a quick (10-20 min) Hangouts call or something? I think we all need to sync up.

I am `PST`

**Comment by saiichihashimoto** [comment]
Yeah... this seems like it needs just a quick hash out session. I'm in `PST` as well. Maybe during the day tomorrow sometime will work for everyone?

**Comment by jdorfman** [comment]
I'm open after 1pm pst. @bnvk how is your day looking?

**Comment by saiichihashimoto** [comment]
1pm sounds good for me!

**Comment by saiichihashimoto** [comment]
Oh no, you meant today! I missed the ball on this, sorry about that.

**Comment by saiichihashimoto** [comment]
Hey sorry everyone, I really dropped the ball on this. Let's finds another time and I'll have to make sure I make it.

**Comment by jdorfman** [comment]
@saiichihashimoto no ball was dropped. Still waiting to hear back from @bnvk

**Comment by jdorfman** [comment]
@saiichihashimoto having trouble getting interest in this project. I don't have the authority to push forward. Closing for now. =/

**Comment by simonv3** [comment]
Hey @jdorfman, do you mean you don't have authority from our side? If so, please don't let any other member of OSD hold you back from doing stuff you think needs to happen. We try to operate without hierarchy because we don't want that expectation on anyone, but this means that if anyone wants something to happen, they have to do it. This can make it a bit tricky to get things done, but it means that the best way to do something is to just forge forward and take the authority! 

For what it's worth you have my full 👍 .

**Comment by jdorfman** [comment]
@simonv3 yeah I want @bnvk to be onboard but I think he is swamped with other stuff, which is fine. Everyone's time is limited.

**Comment by simonv3** [comment]
I'll quote him from above: 

> Anywho, up for the existing maintainers and you to figure out 😁

I reckon he's onboard :)

**Comment by gillisig** [comment]
@simonv3 @jdorfman: Push on! If you feel like you are taking something away from @bnvk then I wouldn't worry about it. If I know him correctly he is all for it.

**Comment by elioqoshi** [comment]
Yep, knowing Brendan he is all up for it. Would be cool to see this happening

**Comment by bnvk** [comment]
> Yep, knowing Brendan he is all up for it. Would be cool to see this happening

Yep. Yep. Thanks @gillisig and @elioqoshi you two do indeed know me well =)

Thanks @jdorfman for taking iniative on this. Apologies for not being more
available at that time!

**Comment by jdorfman** [comment]
@bnvk all good can we reopen?

**Comment by bnvk** [comment]
> @bnvk all good can we reopen?

@jdorfman Yah totally. In the future don't worry if you *"have the authority to 
push forward"* if it's new, creative, and involves *doing* something, just do
it =)

If it is something large and significantly alters existing work (like
entirely re-designing the homepage of the site) then get some consensus!

I believe others are on the same page about this, but please correct if me
I'm wrong :)

**Comment by saiichihashimoto** [comment]
OK, cool. So what's next here? Let's get this train where it needs to go.

**Comment by jdorfman** [comment]
@saiichihashimoto let plan for after the new year. I have projects I need to wrap up at work.

**Comment by bnvk** [comment]
Just pinging @jdorfman and @saiichihashimoto if there is still any interest in this... we just had our first OSD Summit (in Berlin) and are ramping up with some website improvements. Would be cool to see this pick up steam again!!!

**Comment by saiichihashimoto** [comment]
Yeah, it might be interesting to see where this could go. Let's see what the rest of @kogg thinks as as well: @cameronrohani @marcomarandiz

**Comment by jdorfman** [comment]
I totally dropped the ball on this! Unfortunately, SustainOSS and BootstrapCDN are taking up most of my Open Source time so I will have to bow out on this. I think @elioqoshi would be a good leader for this project. Since I use instantlogosearch.com multiple times a week, I will be happy to do any beta testing. =)

Thanks!

**Comment by elioqoshi** [comment]
I am also quite caught up personally, but I think we at Ura can free up some time for this, let me get back on this

**Comment by saiichihashimoto** [comment]
Is this repo still something that's happening? :-)

**Comment by elioqoshi** [comment]
I'd like to get involved, but then again I'm too caught up with identihub.co
Maybe we could deploy it on Identihub rather than fork it....

**Comment by simonv3** [comment]
I was actually thinking that would make a lot of sense.

**Comment by elioqoshi** [comment]
The bash project is up and running after @jdorfman asked me:
https://demo.identihub.co/project/bash#/

We could do a similar thing with a single project having all assets.

Obviously this needs a lot of tweaks to work in a similar fashion but we will get there.
